### DISCUSSION TITLE
Keeping various versions for a framework

### DISCUSSION THREAD

**Discussion by zahidcakici** [the-benchmarker/web-frameworks#7907] [Ideas]
As I understand it, we currently update our framework versions whenever a new release is available, often through dependency bots.

Wouldn’t it be beneficial if we could maintain certain frameworks in multiple versions (at least LTS) to allow for comparison? This approach could help inform upgrade decisions for existing projects.

For example, it would be useful to compare **.NET 8** (LTS) with **.NET 9** (STS) rather than automatically switching to .NET 9 upon release, as many teams prefer to stick with LTS versions.

While this approach may involve additional costs, I believe it’s at least worth discussing

**Comment by waghanza** [comment]
Hi @zahidcakici,

Thanks for your suggestion, but I am opposed to that (at least for now).

Let me explain the rational fehinds :
+ The purpose of this project is to show how frameworks could react, implictly with the safest implementation (e.g : production ready) impliying the safest version of language (e.g : LTS if available)
+ It could increase cost (either maintanance and financial)
+ It complicate to much results

However, it is totally doable to compare frameworks with two version of languages, but this is objective we, imho, could have after having the first objective complete.
